src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" 
+      class="form">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="label">
+          <div class="surname"> 
+            Surname:
+            <input type="text" name="surname" 
+              autocomplete="off"
+            >
+          </div>
+        </label>
+        <label class="label">
+          <div class="name">
+            Name:
+            <input type="text" name="name" 
+              autocomplete="off"
+            >
+          </div>
+        </label>
+        <label class="label">
+          <div class="age"> How old are You?
+            <input type="number" value="12" 
+              min="1" max="100" 
+              name="age"
+            >
+          </div>
+        </label>
+        <label class="label">
+          <div class="birth">
+            Full date of birth:
+            <input type="date" name="date">
+          </div>
+        </label>
+        <label class="label">
+          <div class="accpet">
+            I accept the term of the agreement
+            <input type="checkbox" name="accept" 
+              required
+            >
+          </div>
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="label">
+          <div class="mail">
+            E-mail:
+            <input type="email" placeholder="email@example.com" 
+              name="email"
+            >
+          </div>
+        </label>
+        <label class="label">
+          <div class="password">
+            Password:
+            <input type="password" name="password">
+          </div>
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          <div class="animals">
+            Do you love cats?
+            <input type="radio" name="animal" 
+              value="yes"
+            > Yes
+            <input type="radio" name="animal" 
+              value="no"
+            > No
+          </div>
+        </label>
```

Comment:
Only both inputs should have label wrappers. _Do you love cats?_ should be outside a label